release/relrhcos/rhcos.go

```diff
@@ -0,0 +1,14 @@
+package relrhcos
+
+// Extensions is data specific to Red Hat Enterprise Linux CoreOS
+type Extensions struct {
+	AzureImage *AzureImage `json:"azure-image,omitempty"`
```

Comment:
Since this is a disk image rather than a VM image, would it make sense to use a word other than `image` for clarity?